### PR TITLE
[ROSBE-UNIX] Inherit user's .bashrc file

### PR DESCRIPTION
**Pull request by Extravert-ir**
And add some fanciness

### CODE REVIEW

File: RosBE-Unix/Base-i386/scripts/RosBE-rc [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# extend user's RC file
+source $HOME/.bashrc
```
Comment (ColinFinck) [inline RosBE-Unix/Base-i386/scripts/RosBE-rc]: Please check first if that file exists (using `if [[ -f $HOME/.bashrc ]]`). It may not for people primarily using other shells.
I also hope this move doesn't pull in wild environment variables that screw us later...

File: RosBE-Unix/Base-i386/scripts/RosBE.sh [hunk 2]
```diff
@@ -30,11 +30,6 @@ export BISON_PKGDATADIR="$_ROSBE_ROSSCRIPTDIR/share/bison"
 # The file "RosBE-Version" has been created by the RosBE-Builder.sh script
 export _ROSBE_VERSION=`cat "$_ROSBE_ROSSCRIPTDIR/RosBE-Version"`
 
-# Set the text color
```
Comment (ColinFinck) [inline RosBE-Unix/Base-i386/scripts/RosBE.sh]: Why did you remove that? It's an optional parameter and may be used by people, who want the exact same experience that RosBE-Windows provides ;)

**Review by ColinFinck — CHANGES_REQUESTED** [review]

File: RosBE-Unix/Base-i386/scripts/RosBE-rc (hunk 1 above)
Comment (Extravert-ir) [reply RosBE-Unix/Base-i386/scripts/RosBE-rc]: > I also hope this move doesn't pull in wild environment variables that screw us later
That is something I have in my local installation for several month, so I decided - maybe other people want that too :)

Regarding regression paths, I see one in user's `.bashrc` may have some additions stuff in `$PATH`
